CONTRIBUTING.md

```diff
@@ -0,0 +1,338 @@
+    3. If you cannot find an existing year, month, or date directory that fits your desired published date, you can go ahead and create such directory on your own.
+
+2. Create a markdown file in the directory. Please make sure that the file name is written in English with no capitalized letters. During review session, we may suggest changing it to achieve better SEO performance. (Yes, currently some files names contain capitalized letters, we are working on it, see https://github.com/apache/apisix-website/issues/713)
+
+3. Add text, images, diagrams, charts to the markdown file.
+    1. There is nothing much to say about adding text.
+    2. To add images, please upload images to https://markdown.apiseven.com/#/ , and copy the links from there.
+    3. To add charts or diagrams, we are happy to see any sort of charts and diagrams, as long as it formats correctly on the page. From previous experience, charts with 4 columns or less are the perfect size displayed on screen.
+
+4. Optional: run locally to inspect any typos or formatting issues left behind. We highly recommend running the website locally before submission. Although we implemented lint and error checks in the repository, we suggest running it locally to avoid repeated work. To run it locally, please run `cd website`, and then run `yarn start` commands in your terminal.
+
+5. Submit a pull request to our repository.
+
+#### Blog's header information
+
+Our blogs start with a header like this:
+
+```
+title: "blog's title"
+author: "blog's author"
+authorURL: "author's GitHub ID"
+authorImageURL: "author's GitHub Image"
+keywords:
+- keywords 1
+- keywords 2
+- keywords 3
+- keywords 4
+- keywords 5
+description: description of this blog
+tags: [tag1]
+```
+
+##### title
+
+Required field, the title of the article, the standard expression in the form of title: "title", although the actual test single quotation marks or no quotation marks can be, but in order to maintain the standard, the unified use of English double quotation marks to wrap the text.
+
+##### author
+
+Required field, the article's author name, the standard form of expression for author: "author", although the actual test single quotation marks or no quotation marks can be, but in order to maintain the standard, the unified use of English double quotation marks to wrap the text.
+
+1. community author: if you are willing to provide your real name, use your real name, if not, use your GitHub account name.
+
+2. business case authors: real names, such as John Doe.
+
+##### authorURL
+
+For business users or business cases, you can leave this item blank. Because GitHub is personal, and in the case of a business case, the person represents his company rather than himself, it is recommended that you leave this blank to avoid unnecessary controversy. This does not mean leave authorURL: blank, it means just delete the entire line that corresponds to it, including the variable name.
```

Comment:
```suggestion
For business users or business cases, you can leave this item blank. Because GitHub is personal, and in the case of a business case, the person represents his company rather than himself, it is recommended that you leave this blank to avoid unnecessary controversy. This does not mean leave `authorURL` blank, it means just delete the entire line that corresponds to it, including the variable name.
```